README.md

```diff
@@ -1,33 +1,8 @@
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <h3><b>Portfolio Project</b></h3>
+  <h3><b>Portfolio Project Final Movile Version</b></h3>
 
 </div>
 
```

Comment:
- [ ] Why dont you remove these comments below you have in your readme from your readme file to make it more professional :blush: 
    - "_Describe the tech stack and include only the relevant sections that apply to your project._"
    - "_Describe between 1-3 key features of the application._"